polyhost/device/hid_helper.py

```
@@ -181,6 +181,35 @@ def get_console_output(self):
+
+        After a firmware apply the device re-enumerates; the reconnect path
+        rebuilds the whole HidHelper, but in the come-back-up race the console
+        interface can be missing from that enumeration while raw HID already
+        works — remote_console then silently stayed None until a replug
+        (field 2026-07-05: "no console log after flashing/apply"). Returns
+        True when a console handle is open afterwards."""
+        if self.remote_console is not None:
+            try:
+                self.remote_console.close()
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)